examples/notebooks_new_field/introduction.ipynb

```diff
@@ -0,0 +1,1099 @@
+    " \n",
+    " The boundary conditions complete the set of equations. We have that $u_{0,j,k}=u_{nx,j,k}=u_{i,nz,k}=0$, according to the Dirichlet null conditions at the bottom and lateral boundaries. Moreover, $u_{i,0,k}=u_{i,1,k}$ due to the zero Neumman boundary condition at the surface and $u_{i,j,0}=u_{i,j,1}=0$ due to the initial conditions. \n",
+    "\n",
+    " This is the basic discretization of the wave equation, to be employed in all notebooks. It will be complemented\n",
+    " by some other terms and / or equations, when considering how to discretize the absorbing boundary conditions\n",
+    " and absorbing boundary layers."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# 1.4 - The standard problem\n",
+    "\n",
+    "In order to simulate the acoustic problem with the several absorbing boundary treatments, we define a standard problem to be used not only in this notebook, but also in the following ones. The spatial domain $\\Omega$ will be defined by the following parameters:\n",
+    "\n",
+    "- $x_{I}$ = 13.65 Km = 13650 m;\n",
+    "- $x_{F}$ = 56.65 Km = 56650 m;\n",
+    "- $z_{I}$ = 0.0 Km;\n",
+    "- $z_{F}$ = 9.92 Km = 9920 m;\n",
+    "\n",
+    "with\n",
+    "\n",
+    "- $\\Delta x$ = 0.050 km = 50m;\n",
+    "- $\\Delta z$ = 0.032 km = 32m;\n",
+    "\n",
+    " The integration will be carried out from \n",
+    "\n",
+    "- $t_{I}$ = 0 s = 0 ms;\n",
+    "\n",
+    "to\n",
+    "\n",
+    "- $t_{F}$ = 20 s = 20000 ms;\n",
+    "\n",
+    "The choice of $\\Delta t$ has to respect the CFL condition:\n",
+    "\n",
+    "\\begin{equation}\n",
+    "{\\Delta t}\\leq\\displaystyle\\frac{h\\nu}{v_{max}},\n",
+    "\\end{equation}\n",
+    "\n",
+    "where $\\nu\\leq\\frac{1}{\\sqrt{2}}$ and\n",
+    "\n",
+    "\\begin{equation}\n",
+    "h = \\min\\{ \\Delta x, \\Delta z \\}.\n",
+    "\\end{equation}\n",
+    "\n",
+    "The term $v_{max}$ is the maximum velocity propagation in $\\Omega$ and is given by:\n",
+    "\n",
+    "\\begin{equation}\n",
+    "v_{max}=\\max_{(x,z)\\in \\Omega}\\left\\vert c(x,z) \\right\\vert.\n",
+    "\\end{equation}\n",
+    "\n",
+    "The choice of $\\Delta t$ respecting the CFL condition defines the number of timesteps $NT$:\n",
+    "\n",
+    "\\begin{equation}\n",
+    "\\Delta t =\\frac{T}{NT}.\n",
+    "\\end{equation}\n",
+    "\n",
+    "In all the simulations we employ $\\nu=0.4 $. In this way, the temporal parameters are given by:\n",
+    "\n",
+    "- $\\Delta t$ $\\approx$ 0.0018 s = 1.8 ms;\n",
+    "- $NT$ = 10938.\n",
+    "\n",
+    " The source $f(x, z, t)$  will be a Ricker source with the following properties:\n",
+    "\n",
+    "- Postion at $x:$ $\\bar{x} = 35150 m = 35.15 Km$;\n",
+    "- Position at $z:$ $\\bar{z} = 32 m = 0.032 Km$;\n",
+    "- Peak frequence: $f_{0} = 5 Hz = 0.005 Khz$;\n",
+    "\n",
+    "The graph of $f(\\bar{x}, \\bar{z}, t)$ is given in the figure below.\n",
```

Comment:
Not sure if this is correct in maths. Maybe it should be expressed just as a time-series as in Devito the src.coordinates are not part of the source data per se

Reply:
Solved.

Reply:
The source fucntion is defined for spatial and temporal variables, but in this case we would like to express the fact that just one spatial positon is used and in relation to the time variable the value of the source changes.